js/aes_gcm.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Subtle-based AES-GCM decryption supported on all browser types.
+ */
+
+/** 
+ * Possibly wrap an object in a Promise.
```

Comment:
"Converts IE11 CryptoOperation type to a Promise. See https://msdn.microsoft.com/en-us/windows/dn280996%28v=vs.71%29"

Reply:
Thanks!